### PR TITLE
feat(localization): update french localization

### PR DESCRIPTION
**Pull request by Sharptsa**
Added full localization to French, except for "Underbelly Descent" and "Damplight Chamber" because I can't find the proper translation on WowHead or other websites and I don't play Retail

### CODE REVIEW

**Comment by Sharptsa** [comment]
@yannlugrin As a fellow French speaker, maybe you could give this a quick proofreading and check if I didn't make any typo? :smile:

File: localization/fr-FR.lua [hunk 1]
```diff
@@ -2,35 +2,35 @@ local L = LibStub("AceLocale-3.0"):NewLocale("BeStride", "frFR")
...
--- L["Bindings.Header"] = ""
--- L["Bindings.Regular"] = ""
--- L["Bindings.Ground"] = ""
--- L["Bindings.Repair"] = ""
--- L["Bindings.Passenger"] = ""
+L["Bindings.Header"] = "BeStride"
+L["Bindings.Regular"] = "Bouton de Monte"
+L["Bindings.Ground"] = "Bouton de Monte Forcer Terrestre"
+L["Bindings.Repair"] = "Bouton de Monte Forcer avec Réparateur"
+L["Bindings.Passenger"] = "Bouton de Monte Forcer avec Passagers"
```
Comment (yannlugrin) [inline localization/fr-FR.lua:28]: A suggestion.

```suggestion
L["Bindings.Ground"] = "Bouton Forcer Monture Terrestre"
L["Bindings.Repair"] = "Bouton Forcer Monture avec Réparateur"
L["Bindings.Passenger"] = "Bouton Forcer Monture avec Passagers"
```

File: localization/fr-FR.lua [hunk 2]
```diff
@@ -41,65 +41,65 @@ L["Zone.Oculus"] = "L'Oculus"
...
+L["Settings.EnableNew"] = "Activer automatiquement les nouvelles montures à l'apprentissage"
+L["Settings.EmptyRandom"] = "Choisir une monture aléatoire si aucune monture utilisable n'est sélectionnée"
+L["Settings.RemountAfterDismount"] = "Remonter immédiatement après être descendu de monture"
+L["Settings.NoDismountWhileFlying"] = "Ne pas descendre de monture en vol. Vous devrez atterir ou (si activé dans les options Blizzard) lancer un sort"
+L["Settings.UseFlyingMount"] = "Utiliser une monture volante même dans les zones où voler n'est pas possible"
+L["Settings.ForceFlyingMount"] = "Forcer l'utilisation d'un monture volante même dans les zones où voler n'est pas possible"
+L["Settings.PrioritizePassenger"] = "Prioriser l'utilisation d'une monture avec passagers en groupe"
+L["Settings.NoSwimming"] = "Ne jamais utiliser de monture aquatique pendant la nage"
+L["Settings.FlyingBroom"] = "Toujours utiliser le Balai volant au lieu d'une monture normale"
+L["Settings.Telaari"] = "Toujours utiliser le Talbuk telaari ou le Loup de guerre loup-de-givre à Nagrand"
```
Comment (yannlugrin) [inline localization/fr-FR.lua:64]:
```suggestion
L["Settings.Telaari"] = "Toujours utiliser le Talbuk Telaari ou le Loup de guerre Loup-de-givre à Nagrand"
```

**Comment by yannlugrin** [comment]
@Sharptsa I made some suggestions and translated a missing string. Thanks to having translated everything, I didn't take the time for that (also because I needed someone to check what I write ^^)

File: localization/fr-FR.lua [hunk 3]
```diff
@@ -41,65 +41,65 @@ L["Zone.Oculus"] = "L'Oculus"
...
+L["Settings.ForceRobot"] = "Toujours utiliser le Golem céleste ou l'Extracteur de bois mécanique si vous avez appris Herboristerie"
+L["Settings.Repair.Use"] = "Utiliser une monture avec réparateur si le joueur en possède une et atteint le seuil de durabilité"
+L["Settings.Repair.Force"] = "Force a Repair mount if player owns one"
```
[the remark this answers is not in the document]
Comment (Sharptsa) [reply localization/fr-FR.lua:67]: I failed my copy/paste while rebasing on this one 😄

**Review by DanSheps — APPROVED** [review]

**Pull request merged**